### PR TITLE
Build: Create checksum_update.txt file with tags

### PR DESCRIPTION
**Pull request by MacJoom**
Pull Request for Issue # .

### Summary of Changes

To make life easier for Release Managers:
Create a checksums_update.txt file with tags for direct use in the update server xml's

### Testing Instructions

In your dev environment do a build

### Actual result BEFORE applying this Pull Request

Only checksums.txt is build

### Expected result AFTER applying this Pull Request

Additional checksums_update.txt is build

### Link to documentations
Please select:
- [ ] Documentation link for docs.joomla.org: <link>
- [x] No documentation changes for docs.joomla.org needed

- [ ] Pull Request link for manual.joomla.org: <link>
- [x] No documentation changes for manual.joomla.org needed

### CODE REVIEW

File: build/build.php [hunk 1]
```diff
@@ -679,21 +679,26 @@ function clean_composer(string $dir)
...
-    $checksumsContent = '';
+    $checksumsContent       = '';
+    $checksumsContentUpdate = '';
 
     foreach ($checksums as $packageName => $packageHashes) {
         $checksumsContent .= "Filename: $packageName\n";
 
         foreach ($packageHashes as $hashType => $hash) {
             $checksumsContent .= "$hashType: $hash\n";
+            if (strpos($packageName,"Update_Package.zip") !== False) {
```
Comment (Quy) [inline build/build.php:692]:
```suggestion
            if (strpos($packageName,"Update_Package.zip") !== false) {
```

Comment (Quy) [inline build/build.php]:
```suggestion
            if (strpos($packageName, 'Update_Package.zip') !== false) {
```

File: build/build.php [hunk 2]
```diff
@@ -679,21 +679,26 @@ function clean_composer(string $dir)
...
-    $checksumsContent = '';
+    $checksumsContent       = '';
+    $checksumsContentUpdate = '';
 
     foreach ($checksums as $packageName => $packageHashes) {
         $checksumsContent .= "Filename: $packageName\n";
 
         foreach ($packageHashes as $hashType => $hash) {
             $checksumsContent .= "$hashType: $hash\n";
+            if (strpos($packageName, 'Update_Package.zip') !== false) {
```
Comment (richard67) [inline build/build.php:692]: Why here only for the zip extension? Shouldn’t that be done for all update packages formats? If so, I would recommend to remove the “.zip” from the search string so we only check for “Update_Package”.

Comment (MacJoom) [reply build/build.php:692]: We only need it for the updates.joomla.org xmls - checksums are there only for the zip file. I don't know any other place where the checksums are needed in this format, but may be i'm wrong.

Comment (wilsonge) [reply build/build.php:692]: I think it’s fine for just zips given that’s all the update server uses

**Pull request merged**